looker/resource_looker_saml_config.go

```diff
@@ -305,6 +305,9 @@ func resourceSamlConfigCreateOrUpdate(ctx context.Context, d *schema.ResourceDat
 			// check for existing ID to ensure we update existing relationship
 			if _, ok := sgw["id"]; ok && sgw["id"].(string) != "" {
 				write.Id = conv.P(sgw["id"].(string))
+			} else {
+				id := fmt.Sprintf("%d", rand.Int()))
```

Comment:
Looks great! There's just one too may brackets here. 
```suggestion
				id := fmt.Sprintf("%d", rand.Int())
```

Reply:
Doh, done before coffee

Reply:
Updated